src/js/components/Anchor/anchor.stories.js

```diff
@@ -19,6 +19,8 @@ storiesOf('Anchor', module)
   ))
   .add('With Text', () => (
     <Grommet theme={grommet}>
-      This is a <Anchor label='link' href='#' /> with text.
```

Comment:
this needs to be a single otherwise it will show like this:

<img width="211" alt="screen shot 2018-10-12 at 12 38 23 am" src="https://user-images.githubusercontent.com/1207250/46854892-251a7400-cdb7-11e8-8bd5-68e34f64abc4.png">